lib/Channel.js

```diff
@@ -223,8 +223,11 @@ class Channel {
     }
     
     getMessages(count = 10) {
+        if(Platform.OS == 'android'){
+            this.initialize()
+        }
         return TwilioIPMessagingMessages.getLastMessages(this.sid, count)
-        .then((messages) => messages.map((message) => new Message(message, this.sid)))
+        .then((messages) => {return messages.map((message) => {return new Message(message, this.sid)})})
```

Comment:
@johndrkurtcom do you know for sure this wasn't working? I believe it's redundant because if you don't specify a `{` in an arrow function it returns by default. Unless you were needing to do multiple lines or something..

```
(param1, param2, …, paramN) => expression
         // equivalent to:  => { return expression; }
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Basic_Syntax